person.rb

```diff
@@ -1,4 +1,7 @@
 class Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rental
```

Comment:
- [ ] Again, we intend to both _access (get)_ as well as _write (set)_`rental`, and therefore should be marked with `attr_accessor`. Kindly ensure to improve upon this aspect.